### ISSUE TITLE
Continue Your OData Exploration with EPM

### ISSUE THREAD

**Issue by CimermanDenis** [sap-tutorials/Tutorials#23384]
Tutorials: https://developers.sap.com/tutorials/odata-02-exploration-epm.html
--------------------------

Sadly, access via WebGui has been disabled on 08.09.2023, see https://blogs.sap.com/2017/12/05/new-sap-gateway-demo-system-available/ .Therefore the first unit of this tutorial is not relevant and I recommend to update it. I was thinking about using Eclipse for checking the hierarchy visually, but you might have a better idea (as Eclipse is slowly but steadily being dropped off by SAP, too).

**Comment by jmmargo** [comment]
@qmacro can you please respond to the user's feedback?  Thank you.

**Comment by qmacro** [comment]
Indeed, the system upon which this tutorial is based ... was removed :-( We will be revisiting the OData tutorials at a future date. Sorry for the inconvenience. I will be removing the current tutorials soon, to avoid confusion.
